tools/autograd/load_derivatives.py

```diff
@@ -71,6 +76,28 @@ def add_view_copy_derivatives(
     return view_copy_differentiability_infos
 
 
+def add_generated_functional_derivatives(
```

Comment:
I generate derivatives for every generated functional `NativeFunction`, just re-using the formula from the existing mutable `NativeFunction`.

autograd won't actually run directly on any of these functional operators in most cases; unless you do a `grad(functionalize(foo))`. Even in the backend LTC/XLA case, I'll be moving the functionalize key below autograd soon (so the functional ops will all get run below autograd).

Reply:
Why is this needed at all?
This is done automatically in https://github.com/pytorch/pytorch/blob/c25bdeea26f95d05281cc4309bc400a0eae0b8a4/torchgen/api/autograd.py#L314-L323 already